### DISCUSSION TITLE
%EB%82%B4%EA%B0%80-%EA%B0%9C%EB%B0%9C%EC%9E%90-%EC%BB%A4%EB%A6%AC%EC%96%B4%EB%A5%BC-%EC%8B%9C%EC%9E%91%ED%95%9C-%EC%9D%B4%EC%9C%A0/

### DISCUSSION THREAD

**Discussion by giscus** [SungbinYang/SungbinYang.github.io#2] [Comments]
# %EB%82%B4%EA%B0%80-%EA%B0%9C%EB%B0%9C%EC%9E%90-%EC%BB%A4%EB%A6%AC%EC%96%B4%EB%A5%BC-%EC%8B%9C%EC%9E%91%ED%95%9C-%EC%9D%B4%EC%9C%A0/

양성빈의 블로그

https://sungbin.kr/%EB%82%B4%EA%B0%80-%EA%B0%9C%EB%B0%9C%EC%9E%90-%EC%BB%A4%EB%A6%AC%EC%96%B4%EB%A5%BC-%EC%8B%9C%EC%9E%91%ED%95%9C-%EC%9D%B4%EC%9C%A0/

**Comment by devmeeple** [comment]
스터디에서 뵙고 간단한 대화를 나눴을 때 궁금한 점이 있었는데 글로 공유해 주셔서 감사합니다. 
재밌게 읽었어요! 

오탈자 제보합니다. 

> 그러던 중에 저희 부모님께서는 저의 이런 모습을 좋게 보지는 앟으셨습니다.

**Reply by SungbinYang** [reply]
오탈자 제보 감사드립니다! 추가적으로 깃도 팔로우 해보았습니다!
